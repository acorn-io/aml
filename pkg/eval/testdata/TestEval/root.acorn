a: {
	a: b: 1
	x: $.a.b
	b: 2
}