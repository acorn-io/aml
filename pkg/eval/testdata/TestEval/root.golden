{
  "a": {
    "a": {
      "b": 1
    },
    "b": 2,
    "x": 2
  }
}